### ISSUE TITLE
v-ripple Error in directive ripple unbind hook: "TypeError: Cannot read property 'click' of undefined"

### ISSUE THREAD

**Issue by paritosh149** [quasarframework/quasar#978]
With Reference to issue #892, I am opening this issue.

I am getting following error after upgrading from v0.14.3 to v0.14.4.

> v-ripple Error in directive ripple unbind hook: "TypeError: Cannot read property 'click' of undefined"

Please add null check for el and el.__qripple before calling below functions in ripple.js 
```
 el.removeEventListener('click', ctx.click, false);
 delete el.__qripple;
```

**Comment by rstoenescu** [comment]
A fix is easy indeed. But I am interested as to why this happens. The "el" cannot be null and "__qripple" property should be ensured by Vue that it is there.

Do you have a reproduction repo I can take a look please? This is really interesting as it might highlight a bug in Vue which needs fixing. Thanks!

**Comment by rstoenescu** [comment]
Fixed in v0.15, so closing this. Discovered the corner case where it failed.

**Comment by paritosh149** [comment]
Thanks. Meanwhile, while browsing the changed files, I found that src/directives/move.js file is removed. Please help me understand this.
